### PR TITLE
Blog about the 2i2c CILogon integration

### LINKED ISSUE #621: [blog] 2i2c CILogon integration
### Summary / idea for post

**One-sentence description**

A summary of how CILogon is used at 2i2c, how the integration works and celebrate the partnership.

**Quick outline**

- the number and type of hubs at 2i2c that use CILogon
- the 2i2c cilogon infrastructure
- how the cilogon API allowed for automating various manual tasks
- upstream oauthenticator improvements?
- how cilogon's promptly supported 2i2c to quickly rotate client secret
- celebrate the parnership

### Tasks and updates

- [x] Post draft:  https://github.com/2i2c-org/2i2c-org.github.io/pull/148 (originally at  https://hackmd.io/@georgianaelena/rkkNYuzpi)
- [x] Blog location:  https://2i2c.org/blog/
- [x] Publish to blog
- [ ] Send out tweets and LinkedIn
- [ ] Send to 2i2c listserv (optional)

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Closes https://github.com/2i2c-org/team-compass/issues/621

todo:
- [x] figure out if a partnership has been officially signed before posting this -> https://github.com/2i2c-org/meta/issues/465

### CODE REVIEW

File: content/blog/2023/cilogon-integration/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## The setup
+
+The setup that 2i2c uses, is based on two important tools, the CILogon administrative client and the JupyterHub CILogonOAuthenticator.
+
+### The CILogon administrative client
+
+The [2i2c administrative client](https://cilogon.github.io/oa4mp/server/manuals/dynamic-client-registration.html) provided by CILogon allowed us to automatically manage the CILogon OAuth applications needed for authenticating into the hub.
+
+For each hub that uses CILogon, we dynamically create an OAuth [client application](https://cilogon.github.io/oa4mp/server/manuals/dynamic-client-registration.html) in CILogon and store the credentials safely, using the script at [cilogon_app.py](https://github.com/2i2c-org/infrastructure/blob/HEAD/deployer/cilogon_app.py). The script can also used for `updating` the callback URLs of an existing OAuth application, `deleting` a CILogon OAuth application when a hub is removed or changes authentication methods, `getting` details about an existing OAuth application, `getting all` existing 2i2c CILogon OAuth applications.
```
Comment (consideRatio) [inline content/blog/2023/cilogon-integration/index.md]: The link isnt a permalink but uses HEAD, and may break over time, go for a permalink here.

**Review by consideRatio — APPROVED** [review]
Looks great, nice work!

There was that todo in the top about verifying contract with cilogon etc, and i figure for this which deploys to the website i better delegate the merge-press back to you or someone experienced with the blog post management.

File: content/blog/2023/cilogon-integration/index.md (hunk 1 above)
Comment (GeorgianaElena) [reply content/blog/2023/cilogon-integration/index.md]: 👍🏼 thanks @consideRatio

**Comment by GeorgianaElena** [comment]
Got confirmation from Jim, so I will go for a merge now 🚀

**Pull request merged**